precompiles/bank/setup_test.go

```diff
@@ -32,7 +32,7 @@ type PrecompileTestSuite struct {
 	network     *network.UnitTestNetwork
 	factory     factory.TxFactory
 	grpcHandler grpc.Handler
-	keyring     testkeyring.Keyring
```

Comment:
i would leave this as it was, when renaming the import we are shadowing the import name with the `keyring` variable below

```suggestion
	keyring     testkeyring.Keyring
```